README.md

```diff
@@ -256,6 +256,38 @@ If you are using Coveralls Enterprise and have a self-signed certificate, you ne
 $ goveralls -insecure
 ```
 
+## Prow
+
+Goveralls can be used in [prow](https://github.com/kubernetes/test-infra/tree/master/prow) presubmit jobs, picking up the prow [environment variables](https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#job-environment-variables) `PULL_REFS` and `PULL_NUMBER` to determine the branch and PR number automatically.
+
+To avoid leaking your coveralls repo token you can use the `COVERALLS_TOKEN_FILE` env variable, so that a token gets read and used in the api call.
```

Comment:
```suggestion
To avoid leaking your coveralls repo token you can use the `COVERALLS_TOKEN_FILE` env variable, so that a token gets read and used in the API call.
```